cmd/agent/app/httpserver/collector_proxy_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/uber/jaeger-lib/metrics"
-	mTestutils "github.com/uber/jaeger-lib/metrics/testutils"
+	"github.com/uber/jaeger-lib/metrics/metricstest"
```

Comment:
can we do this in a separate PR?

Reply:
also, why do we need to do this at all? The new jaeger-lib is not released as a version afaik.

Reply:
Fair enough. I opened issue #1140 with PR #1141. It would make sense to do this eventually. Meanwhile, I had originally assumed I was using the rate limiter from jaeger-lib, which prompted me to upgrade the library here.